timescaledb/how-to-guides/install-timescaledb/installation-redhat.md

```diff
@@ -0,0 +1,220 @@
+# Install self-hosted TimescaleDB on Red Hat-based systems
+You can host TimescaleDB yourself on your Red Hat, CentOS, or Fedora system.
+These instructions use the `dnf` package manager on these
+distributions:
+*   Red Hat Enterprise Linux 7
+*   Red Hat Enterprise Linux 8
+*   CentOS 7
+*   CentOS 8
+*   Fedora 33
+*   Fedora 34
+*   Fedora 35
+
+
+<highlight type="warning">
+If you have already installed PostgreSQL using a method other than the `yum` or
+`dnf` package manager, you could encounter errors following these instructions.
+It is safest to remove any existing PostgreSQL installations before you begin.
+If you want to keep your current PostgreSQL installation, do not install
+TimescaleDB using this method.
+[Install from source](/how-to-guides/install-timescaledb/installation-source/)
+instead.
+</highlight>
+
+<procedure>
+
+### Installing self-hosted TimescaleDB on Red Hat-based systems
+1.  At the command prompt, as root, add the PostgreSQL third party repository
+    to get the latest PostgreSQL packages:
+    <terminal>
+
+    <tab label='Red Hat'>
+
+    ```bash
+    dnf install https://download.postgresql.org/pub/repos/yum/reporpms/EL-$(rpm -E %{rhel})-x86_64/pgdg-redhat-repo-latest.noarch.rpm
```

Comment:
I honestly haven't used Red Hat or CentOS for years. Is there a similar issue/need for `sudo` here if users are not root?